atom/browser/browser.h

```diff
@@ -29,6 +29,22 @@ namespace gfx {
 class Image;
 }
 
+#if defined(OS_WIN)
+namespace atom {
+enum class JumpListResult : int;
+}  // namespace atom
+
+namespace mate {
+
+template<>
+struct Converter<atom::JumpListResult> {
```

Comment:
The converters for a type should be placed under the `_api.cc` files that use them, for this one, it should be put in `atom_api_app.cc`.